nixos/modules/services/misc/signald.nix

```diff
@@ -0,0 +1,102 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.signald;
+  dataDir = "/var/lib/signald";
+  defaultUser = "signald";
+in
+{
+  options.services.signald = {
+    enable = mkEnableOption ''
+      start the signald service with the socket at
```

Comment:
The description would now be `Whether to enable start the signald service with the socket` which seems odd.